core/src/main/java/org/apache/carbondata/core/datastore/chunk/reader/dimension/v1/CompressedDimensionChunkFileBasedReaderV1.java

```diff
@@ -99,6 +99,7 @@ public CompressedDimensionChunkFileBasedReaderV1(final BlockletInfo blockletInfo
 
   @Override public DimensionColumnPage decodeColumnPage(
       DimensionRawColumnChunk dimensionRawColumnChunk, int pageNumber) throws IOException {
+    boolean isLongStringColumn = dimensionRawColumnChunk.isLongStringColumn();
```

Comment:
In V1 case it will be always false...as new encoder type will be only supported for V3 format

Reply:
Should we support longStringColumn in V1?

Carbondata now only support writing V1 format.

Reply:
I know, you mean just leave it false?

Reply:
No we support V3 while writing the data...v1/v2 is supported only for reading to support backward compatibility